install.sh

```diff
@@ -15,7 +15,7 @@ CREW_BREW_DIR="${CREW_PREFIX}/tmp/crew/"
 CREW_DEST_DIR="${CREW_BREW_DIR}/dest"
 CREW_PACKAGES_PATH="${CREW_LIB_PATH}/packages"
 
-ARCH="$(uname -m)"
+ARCH="${ARCH:-$(uname -m)}"
```

Comment:
Why would you ever want the `ARCH` environment variable not match the hardware?

Reply:
Using the install.sh script in a x86_64 docker environment with the image files from a i686 machine to get a i686 based clean chromebrew install.

Reply:
The Docker containers I have been using always report the correct architecture with `uname -m`.  If they don't, then there is a problem.